frontend/src/core/comments/components/Comment.css

```diff
@@ -3,6 +3,7 @@
 }
 
 .comments-list .comment a {
+    margin-right: 6px;
```

Comment:
This needs to go into a more specific selector, otherwise it adds margin to the avatar as well.

Also, 4px seems enough.